packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -55,14 +54,14 @@ const verificationDocumentsMapper: TVerificationDocumentsMapper = {
 const JurisdictionCard = ({
     isAdded = false,
     isSelected = false,
-    jurisdiction = 'svg',
+    jurisdiction = Jurisdiction.SVG,
```

Comment:
I dont think we need to pass default value to jurisdiction

Reply:
Updated.